src/pages/Account/Venue/AdvancedDetailsForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useForm, FieldErrors, useFieldArray } from "react-hook-form";
+import { AdvancedVenueInput } from "api/admin";
+
+interface AdvancedDetailsProps {
+  register: ReturnType<typeof useForm>["register"];
+  control: ReturnType<typeof useForm>["control"];
+  values: Partial<AdvancedVenueInput>;
+  errors: FieldErrors<Partial<AdvancedVenueInput>>;
+}
+
+export const AdvancedDetailsForm: React.FC<AdvancedDetailsProps> = (props) => {
```

Comment:
Should this be a template specific form (or will be?)

Reply:
not for the moment, but maybe in the future. If so, the template would need to be passed as a prop